test/api/searchResult.test.ts

```diff
@@ -0,0 +1,148 @@
+import { createMocks } from 'node-mocks-http';
+import { NextRequest, NextResponse } from 'next/server';
+
+import { GET } from '@/app/api/v1/searchResults/route';
+import { createManySearchResults } from '@/repositories/searchResult';
+import { createUser } from '@/repositories/user';
+
+import { user } from '../factories/user';
+import { searchResult } from '../factories/searchResult';
+
+describe('Search result get API:', () => {
```

Comment:
Keep it simple with:

```suggestion
describe('GET api/v1/search-results', () => {
```